mpd/mpd.go

```diff
@@ -81,13 +81,48 @@ type Period struct {
 	AdaptationSets  []*AdaptationSet `xml:"AdaptationSet,omitempty"`
 }
 
+type DescriptorType struct {
+	SchemeIDURI *string `xml:"schemeIDURI,attr"`
+	Value       *string `xml:"value,attr"`
+	ID          *string `xml:"id,attr"`
+}
+
+// ISO 23009-1-2014 5.3.7
+type CommonAttributesAndElements struct {
+	Profiles                  *string               `xml:"profiles,attr"`
+	Width                     *string               `xml:"width,attr"`
+	Height                    *string               `xml:"height,attr"`
+	Sar                       *string               `xml:"sar,attr"`
+	FrameRate                 *string               `xml:"frameRate,attr"`
+	AudioSamplingRate         *string               `xml:"audioSamplingRate,attr"`
+	MimeType                  *string               `xml:"mimeType,attr"`
+	SegmentProfiles           *string               `xml:"segmentProfiles,attr"`
+	Codecs                    *string               `xml:"codecs,attr"`
+	MaximumSAPPeriod          *string               `xml:"MaximumSAPPeriod,attr"`
```

Comment:
Why is the XML attr for this capitalised (but the others aren't)?

Reply:
?? can you come point this out I'm being blind.

Reply:
`xml:"MaximumSAPPeriod,attr` instead of `xml:"maximumSAPPeriod,attr`

Reply:
good spot, I couldn't see that 👓  👍